test/cache_test.rb

```diff
@@ -150,6 +156,14 @@ def test_fragment_fetch_with_virtual_associations
       assert_equal({ place: 'Nowhere' }, ActionController::Base.cache_store.fetch(@location.cache_key))
     end
 
+    def test_fragment_cache_with_inheritance
+      inherited = render_object_with_cache(@role, serializer: InheritedRoleSerializer)
+      base = render_object_with_cache(@role)
+
+      assert_includes(inherited.keys, :special_attribute)
```

Comment:
You're right. That's annoying asser_equal is reverse order
On Mon, Mar 28, 2016 at 9:41 PM Logan Serman <notifications@github.com>
wrote:

> In test/cache_test.rb
> <https://github.com/rails-api/active_model_serializers/pull/1622#discussion_r57665561>
> :
>
> > @@ -150,6 +156,14 @@ def test_fragment_fetch_with_virtual_associations
> >        assert_equal({ place: 'Nowhere' }, ActionController::Base.cache_store.fetch(@location.cache_key))
> >      end
> >
> > +    def test_fragment_cache_with_inheritance
> > +      inherited = render_object_with_cache(@role, serializer: InheritedRoleSerializer)
> > +      base = render_object_with_cache(@role)
> > +
> > +      assert_includes(inherited.keys, :special_attribute)
>
> Not sure what you mean here, should I not use assert_includes? Or am I
> misunderstanding this method?
> http://apidock.com/ruby/v1_9_3_392/MiniTest/Assertions/assert_includes
>
> —
> You are receiving this because you were mentioned.
>
>
> Reply to this email directly or view it on GitHub
> <https://github.com/rails-api/active_model_serializers/pull/1622/files/d0389ca765353add4b55d699a17452234ba42cd0#r57665561>
>